helm/crds/eventlogger.bakito.ch_eventloggers.yaml

```diff
@@ -51,6 +51,29 @@ spec:
                   type: string
                 minItems: 0
                 type: array
+              imagePullSecrets:
+                description: |-
+                  ImagePullSecrets is an optional list of references to secrets in the same namespace to use for pulling any of the images used by this EventLoggerSpec.
+                  If specified, these secrets will be passed to individual puller implementations for them to use.
+                items:
+                  description: |-
+                    LocalObjectReference contains enough information to let you locate the
+                    referenced object inside the same namespace.
+                  properties:
+                    name:
+                      default: ""
+                      description: |-
```

Comment:
Is there a possibility to have proper descriptions generated here?
The TODO's do not look so pretty.

Reply:
It is automatically generated by the controller-gen

Reply:
I'll check if I can fix it after generation